plugins/node/opentelemetry-instrumentation-express/README.md

```diff
@@ -58,9 +58,10 @@ Express instrumentation has few options available to choose from. You can set th
 
 | Options | Type | Example | Description |
 | ------- | ---- | ------- | ----------- |
-| `ignoreLayers` | `IgnoreMatcher[]` | `[/^\/_internal\//]` | Ignore layers that by match. |
-| `ignoreLayersType`| `ExpressLayerType[]` | `['request_handler']` | Ignore layers of specified type. |
-| `spanNameHook` | `SpanNameHook` | `() => 'my-span-name'` | Can be used to customize span names by returning a new name from the hook. |
+| `ignoreLayers`       | `IgnoreMatcher[]`    | `[/^\/_internal\//]` | Ignore layers that by match. |
+| `ignoreLayersType`   | `ExpressLayerType[]` | `['request_handler']` | Ignore layers of specified type. |
+| `spanNameHook`       | `SpanNameHook`       | `() => 'my-span-name'` | Can be used to customize span names by returning a new name from the hook. |
+| `spanAttributesHook` | `SpanNameHook`       | `() => { 'key': 'value' }` | Can be used to customize span attributes by returning a new list of attributes from the hook. |
```

Comment:
[I think that the value `() => { 'key': 'value' }` is not valid in this context](https://stackoverflow.com/questions/28770415/ecmascript-6-arrow-function-that-returns-an-object), as `{` is interpreted as function's body.

You should use parentess like this:
```js
`() => ({ 'key': 'value' })`
```